src/App.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
Next time write the components on different jsx
its not good practice to write all code on app js